back/app/policies/phase_policy.rb

```diff
@@ -35,6 +35,10 @@ def index_xlsx?
     survey_results?
   end
 
+  def index_mini?
+    true
```

Comment:
I'm still thinking about this. `true` is probably bad, as it means anyone can hit the endpoint and get all phases.